modules/fuse-credential-store/fuse-credential-store-commands/src/main/java/org/jboss/fuse/credential/store/command/CreateCredentialStore.java

```diff
@@ -136,6 +136,7 @@ public Object execute() throws Exception {
                 System.err.println("Passwords do not match.");
                 return null;
             }
+            password = password1;
```

Comment:
Did PME change this?

Reply:
fortunately no ;) My mistake.

Reply:
I didn't fully test non-batch scenario. Or rather didn't check this part when adding batch support.